module/exporter/src/Infrastructure/Handler/Export/ProcessExportCommandHandler.php

```diff
@@ -8,39 +8,34 @@
 
 namespace Ergonode\Exporter\Infrastructure\Handler\Export;
 
-use Ergonode\Exporter\Domain\Repository\ExportRepositoryInterface;
-use Ergonode\Exporter\Infrastructure\Provider\ExportProcessorProvider;
 use Ergonode\Exporter\Domain\Command\Export\ProcessExportCommand;
-use Ergonode\Product\Domain\Repository\ProductRepositoryInterface;
-use Ergonode\Exporter\Domain\Repository\ExportLineRepositoryInterface;
 use Doctrine\DBAL\DBALException;
-use Ergonode\Channel\Domain\Repository\ChannelRepositoryInterface;
 
 class ProcessExportCommandHandler
 {
-    private ExportRepositoryInterface $exportRepository;
```

Comment:
It would be good to check why it is commented on in the code before removing it.

Reply:
As above though it's a bit different commit.